core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,25 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, cursor):
+        """Generates a unique id for the training job of the form
+        {{exp_id}}.{{random_hash_of_16_chars}}
+
+        Args:
+            include_deleted: bool. If True, then entities that have been marked
+                deleted are returned as well. Defaults to False.
+        Returns:
+            List of the ClassifierTrainingJobModels with status new or pending.
+        """
+        query = cls.query(cls.status.IN([
+            feconf.TRAINING_JOB_STATUS_NEW,
+            feconf.TRAINING_JOB_STATUS_PENDING]))
+        query = query.order(cls.created_on, cls.id)
+        job_models = query.fetch(10, start_cursor=cursor)
```

Comment:
This can be `job_models, cursor, more = query.fetch_page(10, start_cursor=cursor)` as well (check [here](https://cloud.google.com/appengine/docs/standard/python/ndb/queries#cursors)). `more` signifies whether there are more pages available or not.